waitForMongo.js

```diff
@@ -0,0 +1,31 @@
+// 
```

Comment:
I would put this in a `scripts` subdirectory like Evergreen does to avoid polluting the top-level namespace too much.

Reply:
Fixed